examples/tv-casting-app/linux/simple-app-helper.cpp

```diff
@@ -49,8 +49,10 @@ void DiscoveryDelegateImpl::HandleOnAdded(matter::casting::memory::Strong<matter
     if (commissionersCount == 0)
     {
         ChipLogProgress(AppServer, "Select discovered Casting Player (start index = 0) to request commissioning");
+        ChipLogProgress(AppServer, "Include the cgp flag to attempt the Commissioner-Generated Passcode commissioning flow");
 
-        ChipLogProgress(AppServer, "Example: cast request 0");
+        ChipLogProgress(AppServer, "Example1 Commissionee Passcode: cast request 0");
+        ChipLogProgress(AppServer, "Example2 Commissioner Passcode: cast request 0 cgp");
```

Comment:
in the future, we might want commissioner passcode to be the default

Reply:
Noted. We'll also need corresponding changes on the tv-app side since now the tv-app auto-commissions using the default passcode upon "tv-app shell> controller ux ok".